### PR TITLE
Removed missed merge text and made terminology consistent

### PR DESCRIPTION
**Pull request by abuehrle**
There was some merge text on the install page that I have deleted @2opremio

### CODE REVIEW

File: site/introducing.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Controlling Scope containers goes beyond simple lifecycle operations. Scope can
 
 Weave Scope can visualize your app within:
 
- * Docker Datacenter
+ * Docker Datacenters
```
Comment (2opremio) [inline site/introducing.md]: Why? Docker Datacenter is a product https://www.docker.com/products/docker-datacenter

Comment (abuehrle) [inline site/introducing.md]: Didn't realize that.  I didn't know what that was tbh. Will change that.

File: site/how-it-works.md [hunk 2]
```diff
@@ -70,12 +70,9 @@ Scope can also be used to feed reports to Weave Cloud. With Weave Cloud, you can
 
 To register with the service, go to [https://cloud.weave.works](https://cloud.weave.works), enter your email address and wait for the confirmation email that contains your service token.
 
-![`Weave Cloud` Signup](images/weave-cloud-signup.png)
+The cloud token is accessible from the Weave Cloud main page after you've logged in:
```
Comment (2opremio) [inline site/how-it-works.md]: This won't be true very soon, once we rework the landing page. Can you be unspecific about it? (or mention the Scope/Explore section)

File: site/introducing.md (hunk 1 above)
Comment (2opremio) [inline site/introducing.md]: As a general recommendation: please look up things before changing them :)

File: site/how-it-works.md (hunk 2 above)
Comment (abuehrle) [inline site/how-it-works.md]: I will wait for that change, since this is effects _all of the docs_ and not just scope.

Comment (2opremio) [inline site/how-it-works.md]: OK, your call

**Pull request merged**